modules/data.atmosphere/R/met.process.R

```diff
@@ -25,7 +25,7 @@ met.process <- function(site, input, start_date, end_date, model, host, bety, di
   dbparms <- list(driver=driver, user=user, dbname=dbname, password=password, host=bety.host)
   con       <- db.open(dbparms)
   
-  met <- input$met$type
+  met <- input$met$met.source
```

Comment:
source is a function in R and so I don't think it can be used as a variable name.